generated/python/grpc-google-cloud-datastore-v1/google/cloud/grpc/datastore/v1/datastore_pb2_grpc.py

```diff
@@ -1,157 +0,0 @@
-import grpc
-from grpc.framework.common import cardinality
-from grpc.framework.interfaces.face import utilities as face_utilities
```

Comment:
Using `git mv` when doing PRs like this with changing package names or code paths can make the diffs easier to read. Just a thought for future such PRs.